src/app.js

```diff
@@ -32,10 +33,23 @@ app.use(favicon(path.join(app.get('public'), 'favicon.ico')));
 // Host the public folder
 app.use('/', feathers.static(app.get('public')));
 
+
 // Set up Plugins and providers
 app.configure(hooks());
 app.configure(mongodb);
-app.configure(rest());
+
+  app.configure(rest(function(req, res) {
+    // https://docs.feathersjs.com/api/rest.html
+    function json() {
+      res.end(JSON.stringify(res.data));
+    }
+    res.format({
+    'application/vnd.api+json': function(){convertToJsonapi(req, res)},
+    'application/json': json,
+    'default': json,
```

Comment:
Please have a look at the lines before this comment: default is json. I do not know why it does not choose this.

Reply:
Would you like to request using application/json in the server?

Reply:
What do you think could be a solution?

I could also check the endpoint in the convertToJsonapi function and only use this conversion for the /v1 requests.

Reply:
hm. that sounds good to me.

Reply:
is it ok if I work on the search branch? #36

Reply:
Regarding [res.format](http://expressjs.com/en/4x/api.html#res.format):

> however if it is “\*/\*”, then the response will be “hey”

in 

```javascript
res.format({
  'text/plain': function(){
    res.send('hey');
  },

  'text/html': function(){
    res.send('<p>hey</p>');
  },

  'application/json': function(){
    res.send({ message: 'hey' });
  },

  'default': function() {
    // log the request and respond with 406
    res.status(406).send('Not Acceptable');
  }
});
```

I think the default behaviour from curl etc. is, that no accept header means `*/*`.